app/bundles/LeadBundle/Helper/TokenHelper.php

```diff
@@ -95,13 +98,40 @@ private static function getTokenValue(array $lead, $alias, $defaultValue)
             $value = $lead['companies'][0][$alias];
         }
 
-        if ('true' === $defaultValue) {
-            $value = urlencode($value);
-
+        if ($value) {
+            switch ($defaultValue) {
+                case 'true':
+                    $value = urlencode($value);
+                    break;
+                case 'datetime':
+                case 'date':
+                case 'time':
+                    $dt   = new DateTimeHelper($value);
+                    $date = $dt->getDateTime()->format(
+                        (new ParamsLoaderHelper())->getParameters()['date_format_dateonly']
+                    );
+                    $time = $dt->getDateTime()->format(
+                        (new ParamsLoaderHelper())->getParameters()['date_format_timeonly']
+                    );
+                    switch ($defaultValue) {
+                        case 'datetime':
+                            $value = $date.' '.$time;
+                            break;
+                        case 'date':
+                            $value = $date;
+                            break;
+                        case 'time':
+                            $value = $time;
+                            break;
```

Comment:
@kuzmany we just found out that this change disabled the default value that exists for all other fields. Do you think you could add 
```php
default:
$value = $defaultValue;
```
? I think that would got back the original behavior as well as the new one with 3 special keywords for formatting. What do you think?

Reply:
@escopecz default value should work. This line should cover it https://github.com/kuzmany/mautic/blob/3ab07782dc8c549b48d16876f40a940997eba438/app/bundles/LeadBundle/Helper/TokenHelper.php#L133

Reply:
I missed that. Thanks for looking into it!